packages/dapp/src/components/listinglist/ListingListItemApolloContainer.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import { DispatchProp } from "react-redux";
+import { Query } from "react-apollo";
+import {
+  LISTING_QUERY,
+  transformGraphQLDataIntoNewsroom,
+  transformGraphQLDataIntoListing,
+} from "../../helpers/queryTransformations";
+import ListingListItem from "./ListingListItem";
+
+export interface ListingListItemApolloContainerOwnProps {
+  listingAddress: string;
+  even: boolean;
+}
+
+export default class ListingListItemApolloContainerComponent extends React.Component<
+  ListingListItemApolloContainerOwnProps & DispatchProp<any>
+> {
+  public render(): JSX.Element {
+    const listingAddress = this.props.listingAddress;
+    return (
+      <Query query={LISTING_QUERY} variables={{ addr: listingAddress }}>
```

Comment:
Is it not possible to get the item view in the list query, feel like this way is a lot more network traffic then we need